config/sync/block.block.department_details_contact_us.yml

```diff
@@ -1,25 +1,33 @@
-uuid: c36a96d3-6ad0-4b4d-81ed-8fccb8ae30c9
+uuid: e2c1e57e-7a1e-4155-b346-0def4648579e
 langcode: en
 status: true
 dependencies:
   module:
     - dept_core
+    - domain
     - system
   theme:
     - nicsdru_dept_theme
-id: departmentaldetails_2
+id: department_details_contact_us
 theme: nicsdru_dept_theme
 region: content
-weight: -15
+weight: -13
 provider: null
 plugin: dept_core_departmental_details
 settings:
   id: dept_core_departmental_details
-  label: 'Contact us'
-  label_display: visible
+  label: 'Department details - contact us'
```

Comment:
Did you mean for this to revert?

Reply:
Yep - think so ... not sure now!  Will change later if needs be.